gradle.properties

```diff
@@ -11,7 +11,7 @@ android.useAndroidX=true
 android.enableJetifier=false
 
 GROUP=com.andretietz.retroauth
-VERSION_NAME=3.0.0-beta19-SNAPSHOT
+VERSION_NAME=3.0.0-beta20-SNAPSHOT
```

Comment:
```suggestion
VERSION_NAME=3.0.0-beta19-SNAPSHOT
```

Reply:
My versioning is mainly automated, this is why you don't need to increase it. On the next release, the changes will be in. Optionally you can use the snapshot versions, if you need it immediately